exercises/practice/eliuds-eggs/tests/eliuds_eggs_test.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/test;
+
+@test:Config
+function zeroEggs() {
```

Comment:
```suggestion
function testZeroEggs() {
```

Normally we add `test` prefix infront of a test case.

Reply:
Sounds good. I copied the basic setup from grains originally, and that doesn't have the test prefixes. You may want to update those separately for consistency.

Reply:
sure